### PR TITLE
Fixes  #135 #71 govau/uikit#389

### LINKED ISSUE #135: Add Main Nav and Side Nav components to the in progress section.

### PR DESCRIPTION
**Pull request by alex-page**
Fixes #135 #71 govau/uikit#389

- Add `main nav` and `side nav` to in progress
- `Skip link` documentation improved
- Add `search box` to suggestions
- Update react usage for `accordion`
- Add status to the in progress and suggested tables on the components page

### CODE REVIEW

File: content/components/_all.yml [hunk 1]
```diff
@@ -1516,9 +1516,9 @@ table:
   hasSass: true    # <-- auto generated, don't change
   hasJs: false    # <-- auto generated, don't change
   hasReact: true    # <-- auto generated, don't change
-navigation:
-  name: Navigation
-  description: A list of links that can be used to access different sections of the website
+main-nav:
+  name: Main nav
+  description: Horizontal list of links to key areas on the website usually in the header
```
Comment (pattyde) [inline content/components/_all.yml]: Small copy change:

> Horizontal list of links to key areas on the website. Usually located in the header.

File: content/components/_all.yml [hunk 2]
```diff
@@ -1546,16 +1543,94 @@ navigation:
...
+  ID: main-nav    # <-- auto generated, don't change
+  version: 0.0.0    # <-- auto generated, don't change
+  dependencies:    # <-- auto generated, don't change
+    - core
+  hasSass: true    # <-- auto generated, don't change
+  hasJs: false    # <-- auto generated, don't change
+  hasReact: true    # <-- auto generated, don't change
+side-nav:
+  name: Side nav
+  description: A list of links next to the body content
```
Comment (pattyde) [inline content/components/_all.yml]: Could we expand on the description a bit? How is this different from a `link list`?

> A horizontal list of links for site navigation typically placed next to the body content. Can be a better choice than the `link list` component depending on context.

:point_up:  just a suggestion to build on...

File: content/components/skip-link/default.md [hunk 3]
```diff
@@ -21,8 +29,18 @@ code:
...
+      <div id="content" tabindex="-1">
+        Some content here
+      </div>
 ---
 
 Skip links are not visual until they receive focus.
 
 Include skip links between the opening of the `<body>` and the page `<header>`.
+
+The content that the skip links goes to is linked with and `id` and it moves the focus to the element with `tabindex="-1"`.
```
Comment (pattyde) [inline content/components/skip-link/default.md]: Little typo to fix up:

> The content that the skip links goes to is linked with **an** `id` and it moves the focus to the element with `tabindex="-1"`.

Comment (pattyde) [reply content/components/skip-link/default.md]: @alex-page does this description go far enough to resolve https://github.com/govau/uikit/issues/389 ?

Should there be an explicit recommendation to add `tabindex="-1"` to the destination anchor?

File: content/components/_all.yml (hunk 2 above)
Comment (alex-page) [reply content/components/_all.yml]: Looks good but it's vertical

File: content/components/skip-link/default.md (hunk 3 above)
Comment (alex-page) [reply content/components/skip-link/default.md]: I think it does, I updated the examples to copy and paste aswell

File: content/components/_all.yml (hunk 2 above)
Comment (pattyde) [reply content/components/_all.yml]: Yeah... that's what I meant...

Good pickup ;)

**Pull request merged**